torch/_tensor.py

```diff
@@ -1026,6 +1027,61 @@ def __torch_function__(cls, func, types, args=(), kwargs=None):
             else:
                 return _convert(ret, cls)
 
+    def __dlpack__(self, stream=None):
+        """
+        Creates a dlpack capsule of the current tensor to
+        be exported to other libraries.
+
+        This function will be called from the `from_dlpack` method
+        of the consumer library.
+
+        Args:
+            stream (integer or None): A Python integer representing a pointer
+            to a stream (CUDA or ROCm). `stream` is provided by the consumer
+            to the producer to instruct the producer to ensure that operations
+            can safely be performed on the array.
+            The pointer must be a positive integer or
+            -1 . If stream is -1 , the value may be used by the consumer to
```

Comment:
I'm not really sure what -1 means from this description -- is there a documentation pointer?

Reply:
yes, the dlpack documentation covers this https://github.com/rgommers/array-api/blob/main/spec/API_specification/array_object.md#__dlpack__--streamnone